asciidoc/get-started.adoc

```diff
@@ -277,49 +278,65 @@ The release notes for this driver are available https://github.com/neo4j/neo4j-j
 
 ======
 
-// [.include-with-python]
-// ======
-//
-// To find out the latest version of the driver, visit https://pypi.org/project/neo4j/.
-//
-// To install the latest version of the driver:
-//
-// [source, shell, subs="attributes, specialcharacters"]
-// ----
-// pip install neo4j-driver
-// ----
-//
-// It is also an option to install a certain version of the driver.
-//
-// ========
-//
-// The following is the syntax for installing a certain version of the driver.
-//
-// [source, shell, subs="attributes, specialcharacters"]
-// ----
-// pip install neo4j-driver==$PYTHON_DRIVER_VERSION
-// ----
-//
-// In the following example we are installing driver version {python-driver-version}.
-//
-// [source, shell, subs="attributes, specialcharacters"]
-// ----
-// pip install neo4j-driver=={python-driver-version}
-// ----
-//
-// ========
-//
-// The release notes for this driver are available https://github.com/neo4j/neo4j-python-driver/releases[here].
-//
-// ======
+[.include-with-python]
+======
+
+To find out the latest stable version of the Python Driver, visit https://pypi.org/project/neo4j/
+
+To find a list of all available releases, visit https://pypi.org/simple/neo4j/
+
+To install the latest stable version of the Python Driver:
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver
+----
+
+It is also an option to install a certain version of the driver.
+
+Get the latest version, including alphas:
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver>=4.0.0a
+----
+
+Get the latest version, including betas:
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver>=4.0.0b
+----
+
+Get the latest version, including release candidates:
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver>=4.0.0rc
+----
+========
+
+The following is the syntax for installing a certain version of the Python Driver:
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver==$PYTHON_DRIVER_VERSION
+----
+
+In the following example we are installing Python Driver version {python-driver-version}.
+[source, shell, subs="attributes, specialcharacters"]
+----
+pip install neo4j-driver=={python-driver-version}
+----
+
+========
```

Comment:
This section I think should be an example - to generate a reference number, and example title we need to add a line above line 313 as follows:

```.Installation with Python```